### DISCUSSION TITLE
trailing

### DISCUSSION THREAD

**Discussion by Odeneus** [beeb/pancaketrade#82] [Q&A]
Hi i am a little bit confused about the trailing option, i would appreciate if you could break it down a bit to me. Is the function of the trailing stop loss to stop the trade and do nothing when the price hits a certain target or is it more like a trailing limit buy/ sell, where it can buy or sell at a better price depending on how much percentage the price keeps going up or down. For example if i set trailing at 2% and set limit sell at 100, it can sell at 102 or higher, depending on how high the price goes without coming down

**Comment by beeb** [comment]
Yeah more like the second option you mentioned. If you set it for a sell order and the price goes up, it will activate at the limit price and keep tracking as long as it goes up. When there is a drop from the maximum recorded value of more than 2%, it will sell. It's like if the limit were moving up (trailing) together with the price, 2% below it.
